selenium_tests/course_info/course_info_remove_people_tests.py

```diff
@@ -13,7 +13,8 @@ class RemovePeopleTests(CourseInfoBaseTestCase):
     def test_remove_person(self, test_case_id, test_user, canvas_role, role_id):
         """ Removes a user from course using the Admin Console """
 
-        course_instance_id = self.test_settings['test_course']['cid']
+        test_course_key = 'test_course_SB_ILE'
```

Comment:
Fixed here too.  Scanned through all the test files to see if I modified it anywhere else - seems these are the only two where test course had been modified. Thanks!